-1 
Tested at: 2cbce2178c2b8dd86bc293de696b56980a6e64ba
 I found an error when building: 
 
 <pre>>> Compiling edm plugin /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-07-28-1100/src/FWCore/Services/plugins/Module.cc 
>> Compiling edm plugin /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-07-28-1100/src/FWCore/Services/plugins/ResourceEnforcer.cc 
>> Compiling edm plugin /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-07-28-1100/src/FWCore/Services/plugins/ZombieKillerService.cc 
>> Building shared library tmp/slc6_amd64_gcc491/src/FWCore/Version/src/FWCoreVersion/libFWCoreVersion.so
Copying tmp/slc6_amd64_gcc491/src/FWCore/Version/src/FWCoreVersion/libFWCoreVersion.so to productstore area:
/tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-07-28-1100/src/FWCore/Services/plugins/CondorStatusUpdater.cc:82:34: error: 'constexpr' needed for in-class initialization of static data member 'const float edm::service::CondorStatusService::m_defaultEmaInterval' of non-integral type [-fpermissive]
 const float CondorStatusService::m_defaultEmaInterval;
                                  ^
Leaving library rule at FWCore/Version
>> Building binary TestFWCoreServicesDriver
Copying tmp/slc6_amd64_gcc491/src/FWCore/Services/test/TestFWCoreServicesDriver/TestFWCoreServicesDriver to productstore area:
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-10430/191322/summary.html

--- a/FWCore/Services/plugins/CondorStatusUpdater.cc
+++ b/FWCore/Services/plugins/CondorStatusUpdater.cc
@@ -79,7 +79,7 @@ namespace edm {
 using namespace edm::service;
 
 const unsigned int CondorStatusService::m_defaultUpdateInterval;
-const float CondorStatusService::m_defaultEmaInterval;
+constexpr float CondorStatusService::m_defaultEmaInterval;
 
 CondorStatusService::CondorStatusService(ParameterSet const& pset, edm::ActivityRegistry& ar)
   :

